src/meta-srv/src/bootstrap.rs

```diff
@@ -222,6 +230,12 @@ pub async fn metasrv_builder(
                 )?),
             )
         }
+        #[cfg(feature = "pg_kvbackend")]
+        (None, BackendImpl::PostgresStore) => {
+            let pg_client = create_postgres_client(opts).await?;
+            let kv_backend = PgStore::with_pg_client(pg_client).await.unwrap();
+            (kv_backend, None, None)
```

Comment:
It's better to leave a TODO here. We need to implement the leader election for the pg backend.